README.md

```diff
@@ -513,11 +513,11 @@ You can download the script and config file from the repo:
 conda install -c conda-forge rclone
 
 # download the script and rclone config file
-curl https://github.com/USGS-Astrogeology/ISIS3/raw/dev/isis/scripts/downloadIsisData.py -o downloadIsisData.py
+curl -LJO https://github.com/USGS-Astrogeology/ISIS3/raw/dev/isis/scripts/downloadIsisData.py
```

Comment:
I think you want to remove the .py from the end here. Since the script no longer has the .py suffix.

Reply:
yep, thanks